### ISSUE TITLE
Missing file for coco 

### ISSUE THREAD

**Issue by strongwolf** [sfzhang15/RefineDet#117]
I want to know how many epochs did you train for coco using the learning rate 1e-3, 1e-4 and 1e-5 respectively?

**Comment by strongwolf** [comment]
Thank you. I have another question. For input size 512 and 320, did you use the same configs including anchor box and learning schedule? If yes, I want to know in the early epochs, whether the accuracy of 512 net is higher than 320? I think default anchors are suitable for 320 but not for 512 on VOC. Also, did you use the objectness_score when computing loss? I did not see the place you use this parameter in the caffe source code?

**Comment by sfzhang15** [comment]
@strongwolf 
1. Yes, we use the same configs including anchor box and learning schedule. Since we have tried to use one more scale of anchors (i.e., 512) for the input size 512, but the mAP is not improved. The reason may be the two-step regression.
2. We use the objectness_score at [here](https://github.com/sfzhang15/RefineDet/blob/master/src/caffe/util/bbox_util.cpp#L1104) when computing loss, i.e., ignoring easy negatives.
